.changeset/fifty-aliens-rhyme.md

```diff
@@ -0,0 +1,10 @@
+---
+'@hono/eslint-config': major
```

Comment:
Yes. This change includes breaking change, so this is a `major` version up.

Reply:
Feel free to ping me if you need help upgrading other projects that use `@hono/eslint-config`